tests/libraries/token/test_standard_token.py

```diff
@@ -9,11 +9,11 @@ def testerSnapshot(sessionFixture):
     sessionFixture.resetSnapshot()
     sessionFixture.uploadAndAddToController("solidity_test_helpers/StandardTokenHelper.sol")
     standardToken = sessionFixture.contracts['StandardTokenHelper']
-    return sessionFixture.chain.snapshot()
+    return sessionFixture.createSnapshot()
 
 @fixture
 def testStandardTokenFixture(sessionFixture, testerSnapshot):
-    sessionFixture.chain.revert(testerSnapshot)
+    sessionFixture.resetToSnapshot(testerSnapshot)
```

Comment:
There are a handful of other places we follow this pattern.  Feel free to hunt them down and update them as well, possibly in another PR though.

Reply:
Once I understand the .mine() issue I can go hunt and change tests to use new pattern.

Reply:
updated to use new conftest.py helper methods to change the chain.